packages/storage/docs/components/UpdateUserPhotoModal.story.mdx

```diff
@@ -0,0 +1,19 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks';
+import UpdateUserPhotoModal from '../../src/components/UpdateUserPhotoModal';
+import * as stories from '../stories/UpdateUserPhotoModal.story.js';
+
+<Meta title="Modules/Storage/Components" component={UpdateUserPhotoModal} />
+
+# Overview
+
+`UpdateUserPhotoModal` is a component to manage and select images stored in Firebase Storage.
```

Comment:
improve description here

Reply:
@adrianestevez1 this is not solved

Reply:
@LeandroTorresSicilia @adrianestevez1 
Let's use this description ;) 
"UpdateUserPhotoModal is a component for managing the user profile photo. It is built using Firebase, and all images will be stored on Firebase Storage."